jkube-kit/build/api/src/test/java/org/eclipse/jkube/kit/build/api/helper/DockerFileUtilTest.java

```diff
@@ -182,12 +183,8 @@ public void testCustomInterpolation() throws IOException {
     @Test
     public void testMultiStageWithArgs() throws Exception {
         File toTest = copyToTempDir("Dockerfile_multi_stage_with_args");
-        Iterator<String> fromClauses = DockerFileUtil.extractBaseImages(toTest, new Properties(), BuildConfiguration.DEFAULT_FILTER, Collections.emptyMap())
-                .iterator();
-
-        assertEquals("fabric8/s2i-java:latest", fromClauses.next());
-        assertEquals("busybox:latest", fromClauses.next());
-        assertFalse(fromClauses.hasNext());
+        assertThat(DockerFileUtil.extractBaseImages(toTest, new Properties(), BuildConfiguration.DEFAULT_FILTER, Collections.emptyMap()))
```

Comment:
Could we use a String literal instead of the Buildconfiguration.DEFAULT_FILTER constant here?